app/javascript/components/pages/PatientInfo.js

```diff
@@ -29,7 +30,6 @@ export default class PatientInfo extends Component {
       .then((patient) => this.setState({ patient: patient }))
       .catch((errors) => console.log("Medications errors:", errors));
```

Comment:
You might consider later to instead of just logging the errors (use `console.error` also) you might create an error component to show a message so the user knows something went wrong. ice box. You can set the error to state, and if it exists

Reply:
going to look into this for ice box